manifests/server.pp

```diff
@@ -103,8 +107,11 @@
     'standalone': {
       include puppet::server::standalone
     }
+    'server': {
+      include puppet::server::server
+    }
     default: {
-      err('Only "passenger", "thin", and "unicorn" are valid options for servertype')
+      err('Only "passenger", "thin", "serverr", and "unicorn" are valid options for servertype')
```

Comment:
Side note -

The [puppetlabs-hocon](https://github.com/puppetlabs/puppetlabs-hocon) module appears to be intended as the standard way to manage hocon-format configuration files in puppet server and other trapperkeeper projects. It provides approximately the same functionality as the ini_setting resource from puppetlabs/inifile.